src/app/components/PageTitleConfirmation.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { hiddenButAccessible } from '../theme';
+import { AppState } from '../types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+
+function PageTitleConfirmation({
+  className,
+  title,
+}: {
+  className: string;
+  title: string;
+}) {
+  const skipFirst = React.useRef(true);
+  const message = React.useMemo(() => {
+    if (!title) {
+      return '';
+    }
+    if (skipFirst.current) {
+      skipFirst.current = false;
+      return '';
+    }
+    return `Loaded page ${title}`;
+  }, [title]);
+
+  return (
+    <div id='page-title-confirmation' className={className} aria-live='polite'>
+      {message}
+    </div>
+  );
+}
+
+export default connect(({ head: { title } }: AppState) => ({ title }))(styled(
```

Comment:
there is a selector for the title https://github.com/openstax/rex-web/blob/main/src/app/head/selectors.ts#L19

using it will affect nobody's life but it follows the convention. when you have derived values in selectors it does make a difference because they're memoized based on the input state and will help reduce react re-renders

Reply:
I added a auto-focus to main when its content changes, and also revised this to use the selector.